### PR TITLE
The Future is Now

### PR DESCRIPTION
**Pull request by ppoulsen**
Patch Notes:
 - Increased mantle interconnectivity
 - Stoked the flames
 - Fixed - some issues running MantleDash on Windows XP. Windows XP is still not officially supported.
 - Bug fixes

### CODE REVIEW

File: static/terminal.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      t.echo("user username");
+    }
+  };
+
+  // Initialize terminal
+  t = $('#terminal').terminal(commands, {
+    greetings: greetings,
+    name: 'MantleDash_Terminal',
+    height: 150,
+    prompt: "i4o> "
```
Comment (michaelhollman) [inline static/terminal.js]: Hmmm... I think I'd prefer something like `md> ` or `mantledash> `.

**Comment by michaelhollman** [comment]
> Windows XP is still not officially supported.

I have an apple on my desk. I don't know why I stole it from Selleck. I also at a peanut butter rice crispy thingy and it was waaaay too sugary.

**Comment by NathanDeMaria** [comment]
+1 to making the prompt "md>"
It should be noted that I didn't actually read the code yet.  I just saw that this was happening and got excited

**Comment by michaelhollman** [comment]
Is it possible to configure default arguments for the commands? It'd be nice to have text default to empty string and size default to something reasonable for the `gif` command. And maybe default to `info` on the `alert` command.

**Pull request merged**